Twisted-Tutorial.ipynb

```diff
@@ -358,6 +515,13 @@
    ]
   },
   {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Twisted also has in-process time-based loops. This allows integrating periodic tasks in the same application, simplifying deployment. For example, this can be used for cache flush or clearing, for log rotation and other similar purposes."
```

Comment:
"similar purposes" could be more precise.  Maybe:

"...other, similar recurring maintenance tasks"